kernel/inferCumulativity.ml

```diff
@@ -77,7 +77,7 @@ let infer_sort cv_pb variances s =
   | CUMUL ->
     LSet.fold infer_level_leq (Universe.levels (Sorts.univ_of_sort s)) variances
 
-let infer_table_key infos variances c =
+let infer_table_key _infos variances c =
```

Comment:
If info isn't used then we shouldn't pass it.

Reply:
Dunno, just move the code and fixed the warnings. I don't think it should be part of this PR to further modify this code.